### PR TITLE
Using vswhere to get build env

### PR DESCRIPTION
**Pull request by buldo**
Original scripts supports only VS Community. But with vswhere scripts can use any modern VS in any edition

### CODE REVIEW

**Comment by meee1** [comment]
need to remove the exe

**Comment by buldo** [comment]
@meee1 why? As I uderstand from this repo https://github.com/microsoft/vswhere this exe has no dependencies and can be stored in repo.

There is alternative way with using this powershell module https://github.com/Microsoft/vssetup.powershell

I can rewrite current bat scripts to powershell if You allow this

**Comment by meee1** [comment]
its included as part of the installer package on all VS installs
${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer\vswhere.exe

**Comment by buldo** [comment]
@meee1 Fixed

File: build - Lib.ps1 [hunk 1]
```diff
@@ -1,5 +1,10 @@
-$msbuildPath = Split-Path (& "${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer\vswhere.exe" -latest -requires Microsoft.Component.MSBuild -find MSBuild\Current\Bin\amd64\MSBuild.exe | Select-Object -First 1) -Parent
-$env:PATH = $msbuildPath + ';' + $env:PATH
+$installationPath = ./vswhere.exe -prerelease -latest -property installationPath
```
Comment (meee1) [inline build - Lib.ps1:1]: this need fixing

Comment (meee1) [reply build - Lib.ps1:1]: i also dont think this file needed any changes based on what it does

**Pull request closed without merging**